qualtrics_link/migrations/0003_auto_20180105_1238.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2018-01-05 12:38
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('qualtrics_link', '0002_add_acceptance'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='QualtricsUser',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('univ_id', models.CharField(max_length=32)),
+                ('qualtrics_id', models.CharField(max_length=32)),
```

Comment:
Is 32 sufficient for qualtrics_id? Is this diff from qualtrics username?looks like teh length of username is more than 32

Reply:
32 should be sufficient. The username is different than the qualtrics ID. An example ID is: UR_7TEwrqvZr1SaXGz which is only 18 char long